src/lib/utils/options/declaration.ts

```diff
@@ -46,7 +46,7 @@ export class OptionDeclaration {
 
     scope: ParameterScope;
 
-    map: Object;
+    protected map: Object | Map<string, string> | 'object';
```

Comment:
Limiting the scope made it easier to understand where `map` is being used. Is there a reason to keep this value public?